web-console/src/views/load-data-view.tsx

```diff
@@ -175,14 +176,14 @@ const VIEW_TITLE: Record<Stage, string> = {
 
 export interface LoadDataViewProps extends React.Props<any> {
   initSpec: IngestionSpec | null;
-  goToTask: (taskId: string | null, openDialog?: string) => void;
+  taskId?: string | null;
```

Comment:
I would call this `initTaskId` also you need a supervisor Id in case you are coming with a supervisor